src/const.js

```diff
@@ -9,6 +9,7 @@ export const ESLINT_RULE_MAPPING = {
   deprecated_entities: ESLINT_WARNING,
   eval_string_arg: ESLINT_ERROR,
   global_require_arg: ESLINT_WARNING,
+  init: ESLINT_WARNING,
```

Comment:
Again this is too vague - please add some more context to the warning. It would be nice to try and infer a little bit more info from just the error code. `INIT_NULL_FIRST_ARG` or something like that.